Description tags changed; readme added

marcellevn opened the pull request:
- Descriptions for all categories added
- Updated the readme

marcellevn committed:
Description tags changed; readme added

- Descriptions for all categories added
- Updated the readme

marcellevn committed:
Merge branch 'main' into marcellevn-contributor

marcellevn committed:
Last changes made (V1 content)

- Added images
- Did an overall copy checks

suitophobia reviewed (commented):
Can we reduce the amount and size of images for SEO please?

marcellevn committed:
Images updated with compressed files

- Replaced images with smaller file sizes

- Removed 2 x banners that are not as necessary

marcellevn committed:
Removed images not needed right now
